src/olympia/activity/serializers.py

```diff
@@ -37,3 +36,16 @@ def get_action(self, obj):
 
     def get_highlight(self, obj):
         return obj in self.to_highlight
+
+    def get_user(self, obj):
+        """Return minimal user information using ActivityLog.author_name to
+        avoid revealing actual name of reviewers for their review actions if
+        they have set an alias.
+
+        id, username and url are present for backwards-compatibility only."""
+        return {
+            'id': None,
+            'username': None,
+            'url': None,
+            'name': obj.author_name,
+        }
```

Comment:
This will remove id, username and url for all activity log responses, not just reviewer related ones (`self.action in constants.activity.LOG_REVIEW_QUEUE_DEVELOPER`)

Reply:
In theory this serializer is only used by `VersionReviewNotesViewSet` which does that filtering already. But you're right, I should also do the check here just in case, relying on the view to filter is a bad idea (it's a vestige from an older version of this patch that always returned the reviewer alias for all actions)

Reply:
10bc305de5fe220fa4cf7ed75aabeb5070049026. Note that we don't need the extra check for the `name` since `author_name` does it for us.